src/app/modules/perinatal/components/detalle-perinatal.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'detalle-perinatal',
+  templateUrl: './detalle-perinatal.component.html'
+})
+export class DetallePerinatalComponent {
+  carnet;
+  @Input('carnet')
+  set _carnet(value) {
+    this.carnet = value;
+  }
```

Comment:
los Input por algo en especial están tratados así??
si pones  `@Input() carnet : any` y después dentro del componente lo podes usar como this.carnet no te sirve??

Reply:
corregido